city/router.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from dependencies import get_db
+from . import crud, schemas
+
+
+router = APIRouter()
+
+
+@router.post("/cities", response_model=schemas.CityRead)
+async def create_city(city: schemas.CityCreate, db: AsyncSession = Depends(get_db)):
+    return await crud.create_city(db=db, city=city)
```

Comment:
Verify that the `crud.create_city` function is correctly implemented and that it returns an instance of the `City` model that matches the `CityRead` schema.